addons/udes_stock/models/stock_picking_type.py

```diff
@@ -435,6 +435,11 @@ class StockPickingType(models.Model):
         help="Enable/Disable Transfer count button on kanban view",
     )
 
+    # Enable printing
+    u_enable_printing = fields.Boolean(
```

Comment:
Do all the print strategies respect this if not this could cause a lot of confusion.

Reply:
Agree, the name is confusing, because it is only to enable printing of delivery note on mobile, therefore normal strategies ignore this setting.

Reply:
@nocturnalastro @ws-mps I've changed the name, please review.

Reply:
I think it's clear enough now but it should probably be revisited allow the front end to query if a type of printing is enabled. Something like having `printable_report_ids` or something.